Source/WebCore/dom/Document.cpp

```diff
@@ -9718,6 +9718,8 @@ void Document::handlePopoverLightDismiss(const PointerEvent& event, Node& target
 
                     if (!invokerPopover) {
                         if (RefPtr button = dynamicDowncast<HTMLFormControlElement>(*htmlElement)) {
+                            if (RefPtr popover = button->invokeTargetElement(); popover && isShowingAutoPopover(*popover))
```

Comment:
As invokeTargetElement now returns Element we need to dynamicDowncast it to HTMLElement as that's what isShowingAutoPopover() takes.

```suggestion
                            if (RefPtr popover = dynamicDowncast<HTMLElement>(button->invokeTargetElement()); popover && isShowingAutoPopover(*popover))
```